### ISSUE TITLE
App crashes when push notification is received

### ISSUE THREAD

**Issue by vtisnado** [Pushwoosh/pushwoosh-react-native-plugin#17]
Hi,

I follow all the steps and the notification is correctly showed in the status bar, but every time a push is received the app crashes. 

The emulator show an alert saying: **APP_NAME has stopped** and the following error is logged in Crashalytics:

```
Fatal Exception: java.lang.RuntimeException
Unable to instantiate service com.pushwoosh.PushFcmIntentService: java.lang.ClassNotFoundException: Didn't find class "com.pushwoosh.PushFcmIntentService" on path: DexPathList[[zip file "/data/app/MY_APP_PACKAGE/base.apk"],nativeLibraryDirectories=[/data/app/MY_APP_PACKAGE/lib/arm, /data/app/MY_APP_PACKAGE/base.apk!/lib/armeabi-v7a, /system/lib, /vendor/lib, /system/vendor/lib, /product/lib]]
```
I know it crashes in the device because is registered in Crashalytics as well.

I tried removing the DeviceEventEmitter listeners and having the minimal configuration, but nothing fixes the error.

```
Pushwoosh.init({ 
    "pw_appid" : "APP_ID" , 
    "project_number" : "PROJECT_NUMBER" 
  });
  Pushwoosh.register();
```
Do you have any idea about that c**om.pushwoosh.PushFcmIntentService** missing class?

**Comment by wfhm** [comment]
Hi,

Checking it.

**Comment by wfhm** [comment]
Hi,

Could you please share a reproducer with us?

**Comment by 8of** [comment]
Have the exact same crash when trying to open push notification on Android.
`pushwoosh-react-native-plugin: 5.5.4`
I'll try to setup a reproducer later today.

**Comment by wfhm** [comment]
@8of Thank you, it would be highly appreciated!

**Comment by 8of** [comment]
@wfhm sorry for the long wait. I updated original post with more info and reproducer.

**Comment by 8of** [comment]
@wfhm Updated previous post with more info and reproducer.
*UPD* also reworded post to make the point more clear.

**Comment by 8of** [comment]
@vtisnado can you try the fix from my post and tell us if it works for you too.
Or if it's not.

**Comment by vtisnado** [comment]
Thank you @8of I tried something very similar based in this post: https://medium.com/@suchydan/how-to-solve-google-play-services-version-collision-in-gradle-dependencies-ef086ae5c75f and didn't worked for me, but now I delegated the project to @templarstone so he will be updating this issue with the results of the test.

**Comment by templarstone** [comment]
It Work! @8of I am trying with some pushes to android devices and this really work. Now, do you have any result with iOS? Whatever, thank you bro. I keep going testing and I'll inform about it @vtisnado

**Comment by 8of** [comment]
@templarstone 
>Now, do you have any result with iOS?
iOS works fine in our project.

**Comment by wfhm** [comment]
@vtisnado @8of @templarstone The issue is fixed with 5.5.7 release.
